packages/nextcloud/lib/src/helpers/common.dart

```diff
@@ -11,7 +11,9 @@ class VersionCheck {
     required this.versions,
     required this.minimumVersion,
     required int? maximumMajor,
-  }) : maximumMajor = maximumMajor ?? minimumVersion.major;
+    bool? isSupportedOverride,
```

Comment:
I don't see any place where this is `false` and I can't imagine in what use case we'd like to do that.
What about making this non nullable with a default value of `false`.

Reply:
It would be possible to exclude some versions that are known to be bad (e.g. 10.0 works, 10.1 is broken, but 10.2 is fine again and 10.1 could be blocked this way). I'm not planning to implement this at the moment, so I can also change it like you suggested.